specification/dimensions/chargefrequency.md

```diff
@@ -0,0 +1,39 @@
+# Charge Frequency
+
+Charge Frequency indicates how often a charge will occur. Along with the Charge Period column, the Charge Frequency can be used to understand recurrence periods (e.g., monthly, yearly), forecast upcoming charges, and differentiate between one-time and recurring fees for purchases.
+
+The ChargeFrequency column MUST be present and MUST NOT be null or empty. This column is of type String and MUST be one of the allowed values.
```

Comment:
A minor tweak to keep the standard and remove redundancy:
The ChargeFrequency column MUST be present **in the billing data** and MUST NOT **be null**.